QUICK_START_GUIDE.md

```diff
@@ -37,6 +37,7 @@
    ./external/mmdetection/init_venv.sh det_venv
    source det_venv/bin/activate
    pip3 install -e ote_cli/
+   pip3 install -e ote_sdk/
```

Comment:
Does `init_venv.sh` not install `ote_sdk` ?

Reply:
I missed the warning message from init_vens.sh. This line is removed. Thank you!